Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -446,14 +447,37 @@ bool EmbeddedFrag::matchToTemplate(const RDKit::INT_VECT &ringSystemAtoms,
        coordinate_templates.getMatchingTemplates(ringSystemAtoms.size())) {
     // To reduce how often we have to do substructure matches, check ring info
     // and bond count first
-    if (mol->getNumBonds() != rs_mol.getNumBonds()) {
+    if (mol->getNumBonds() != numBonds) {
       continue;
     } else if (mol->getRingInfo()->numRings() != ring_count) {
       continue;
     }
+    // also check if the mol atoms have the same connectivity as the template
+    auto degreeCounts = [DUMMY_ATOMIC_NUM](const RDKit::ROMol &mol) {
```

Comment:
@ZontaNicola I hope this change will fix the last remaining problem: the compile errors on the Mac.

```suggestion
    // also check if the mol atoms have the same connectivity as the template
#ifdef _MSC_VER
    // MSVC++ doesn't like implicitly capturing constexpr variables, this is a bug
    auto degreeCounts = [DUMMY_ATOMIC_NUM](const RDKit::ROMol &mol) {
#else
    // clang generates warnings if you explicitly capture a constexpr variable
    auto degreeCounts = [](const RDKit::ROMol &mol) {
#endif
```